# Direct graph matrix
DGMat: [0, 1, 1, 1, 0, 1, 1, 1, 0]

# Velocity gain
gamma: 0.1

# Sampling time for yaw orientation rate 
Ts: 0.7

# max speed
maxSpeed: 0.5

# use external DP controller
useExtCon: False
kdp: 0.5

useRepel: False
# Force repeling controller gain
kf: 3.5
# Force correction factor
kd: 0.0
# Force vector corr factor
ni: 0.5
# repel radius
rf: 2.0

# Gain matrix
GMat: [0, 0.08, 0.08, 0.08, 0, 0.08, 0.08, 0.08, 0]

# Formation distances matrix
formX: [0, 3, 1.5, -3, 0, -1.5, -1.5, 1.5, 0]
formY: [0, 0, 3, 0, 0, 3, -3, -3, 0]


